langlib/lang.table/src/main/ballerina/table.bal

```diff
@@ -134,6 +224,15 @@ public isolated function forEach(table<MapType> t, @isolatedParam function(MapTy
 #
 # The resulting table will have the same keys as the argument table.
 #
+# ```ballerina
+# table<record {| int id; int salary; |}> employees = table [
```

Comment:
Inconsistent spaces after `{|` and before `|}`.

```suggestion
# table<record {|int id; int salary;|}> employees = table [
```

Please fix wherever applicable.